CMakeLists.txt

```diff
@@ -103,6 +115,9 @@ if(ENABLE_TIOGA)
   add_definitions("-DNALU_USES_TIOGA")
 endif()
 
+find_package(FFTW REQUIRED)
```

Comment:
Thanks for updating the FFTW logic in CMake. These lines just need to be removed, and then I think it looks great.